### DISCUSSION TITLE
Receiving 404 message when calling function which works in DBeaver (and is a renamed copy of an already reachable function)

### DISCUSSION THREAD

**Discussion by MyProxyPass** [PostgREST/postgrest#3479] [Q&A] [answered]
Question to the community -- do I have to do anything to make a function "discoverable" by the PostgREST extension?  I have a call that works (public.f_car_inventory) but then created a copy of the function -- only difference is the name (public.f_car_inventory2) -- even writes to the same underlying table.  Issue is that when calling the new function with the same code that previously worked (again, only updating the name by adding a "2" at the end), I now receive a 404 message.  Is there anything I need to do to make sure the extension can see/interact with the function?  Thank you for recent guidance and I appreciate any more!

import requests
import json

url = f"https://<org>/restapi/v1/rpc/f_car_inventory2"
headers = {"Content-Type": "application/json", "Authorization": f"token"}
data = {"param_make":"honda", "param_model":"pilot", "param_yr":"2011", "param_msrp":"27500"}
json_data = json.dumps(data)

resp = requests.post(url, headers=headers, data=json_data)
print(resp)

**Comment by wolfgangwalther** [comment, marked as answer]
You probably want to read up on the Schema cache and how to reload it: https://postgrest.org/en/v12/references/schema_cache.html
